less/navigation-mobile.less

```diff
@@ -0,0 +1,39 @@
+@media (max-width: 500px) {
+.servicesBody{ 
+    max-width: 500px !important;
+    .navigationBar{
+        z-index: +2 !important;
```

Comment:
You're using important a lot here. Like Dan or someone said "If you're using important, you've got bigger problems".